content/messaging/agent-chat-api/v3.3/rtm-reference/index.mdx

```diff
@@ -4206,17 +4313,41 @@ Informs that a Customer was banned for a specified number of days.
 | **Action**             | `customer_banned` |
 | **Webhook equivalent** | -                 |
 
-### `customer_visit_ended`
+### `customer_transferred`
+
+Informs that the agent no longer has the access to a Customer.
```

Comment:
```suggestion
Informs that a customer is no longer available the agent.
```

Reply:
I'm using the word "available" meaning "accessible". Is that okay or did you use "available" in a different context and I didn't notice? "agent has the access to a Customer" seems as if we were talking about objects, not people. I know that those people translate to data structures, but we can try to can keep it more humanized in the docs :D

Reply:
yup, pretty much "available" == "accessible" in this context